### ISSUE TITLE
Do not use "magic strings" in fragments.py

### ISSUE THREAD

**Issue by gkapfham** [GatorEducator/gatorgrader#146]
**Describe the bug**

The current `fragments.py` file contains source code like this:

```
if counter == 1 and subnode.t == "paragraph" and enter:
                # Initialize paragraph_content
                paragraph_content = ""
                # Stop search for paragraph nodes, as one has been found
                # Instead, start adding content to paragraph_content
                mode_looking = False
```

I think that it would be better if the program did not use words like `"paragraph"` and instead defined variables inside of the function that contained this string. This would better support refactoring in the circumstance in which these strings happen to change and better adhere to the standard that we have already set in this module and other modules.

**Comment by gkapfham** [comment]
It is worth noting that this "magic" string should also be handled by the refactoring of the constant variables.

**Comment by gkapfham** [comment]
Note that this issue is resolved by PR #167.
